scss/mixins/_border-radius.scss

```diff
@@ -1,35 +1,20 @@
 // Single side border-radius
+$radius-side-lists: (
+    all  : (true),
+    top  : (true, true, false, false),
+    bottom  : (false, false, true, true),
+    left  : (true, false, false, true),
+    right  : (false, true, true, false),
+);
 
-@mixin border-radius($radius: $border-radius) {
+// Defaults to all corners at $border-radius setting value with no arguments
+@mixin border-radius ( $size: $border-radius , $side: all ) {
```

Comment:
Expected 0 spaces after `(` instead of ` `<br>Expected 0 spaces before `)` instead of ` `